ginga/rv/plugins/Catalogs.py

```diff
@@ -646,19 +646,19 @@ def filter_results(self, starlist, filter_obj):
         if filter_obj:
             num_cat = len(starlist)
             self.logger.debug("number of incoming stars=%d" % (num_cat))
-            # TODO: vectorize wcs lookup
-            coords = [image.radectopix(star['ra_deg'], star['dec_deg'])
-                      for star in starlist]
-            arr = numpy.array(coords)
-            self.logger.debug("arr.shape = %s" % str(arr.shape))
+            coords = numpy.asarray([(star['ra_deg'], star['dec_deg'])
```

Comment:
I see that you changed `numpy` to `np` in other files. Maybe change it here too for consistency.